requirements/requirements.txt

```diff
@@ -12,6 +12,7 @@ in-place==0.5.0
 jieba==0.42.1
 justext==3.0.1
 lxml_html_clean
+mecab-python3==1.0.9
```

Comment:
Any reason for the hard pin? If not, I'd prefer to just keep it at `mecab-python3`